Python/Product/PythonTools/PythonTools/LanguageServerClient/FileWatcher/Listener.cs

```diff
@@ -169,20 +171,7 @@ class Listener : IDisposable {
                 if (didChangeParams.Changes.Any()) {
                     await _rpcWrapper.NotifyWithParameterObjectAsync(Methods.WorkspaceDidChangeWatchedFiles.Name, didChangeParams);
 
-                    if (renamedArgs != null) {
-                        var textDocumentIdentifier = new TextDocumentIdentifier();
-                        textDocumentIdentifier.Uri = new Uri(renamedArgs.OldFullPath);
-
-                        var closeParam = new DidCloseTextDocumentParams();
-                        closeParam.TextDocument = textDocumentIdentifier;
-                        await _rpcWrapper.NotifyWithParameterObjectAsync(Methods.TextDocumentDidClose.Name, closeParam);
-
-                        var textDocumentItem = new TextDocumentItem();
-                        textDocumentItem.Uri = new Uri(renamedArgs.FullPath);
```

Comment:
`TextDocumentItem` was missing content and version params.
This open/close calls seem to be handled in rename case.  Now we only send file change events above.  `WorkspaceDidChangeWatchedFiles`